<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellDialog</class>
 <widget class="QDialog" name="SellDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Продажа</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/images/favicon.ico</normaloff>:/images/favicon.ico</iconset>
  </property>
  <widget class="QDialogButtonBox" name="DialogBtnBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>245</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="printBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Печать сертификата</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddCustomerBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Новый клиент</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>310</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="StatusBarLayout"/>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>282</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="DurationLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="DurationLabel">
      <property name="text">
       <string>Срок действия (месяцев):</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="DurationSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>32767</number>
      </property>
      <property name="value">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="UnlimitedCheckBox">
      <property name="text">
       <string>Бессрочная</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>181</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="DatesLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="ActivationDateLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="ActivationDateLabel">
        <property name="text">
         <string>Дата активации</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="ActivationDateSpinBox"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QFormLayout" name="ExpirationDateLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="ExpirationDateLabel">
        <property name="text">
         <string>Дата окончания</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="ExpirationDateSpinBox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>220</y>
     <width>291</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="CredentialsLayout">
    <item>
     <layout class="QFormLayout" name="UsernameLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="UsernameLabel">
        <property name="text">
         <string>Имя пользователя</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="UsernameEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="PasswordLayout">
      <item>
       <widget class="QLabel" name="PasswordLabel">
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PasswordEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="EngineerLayout">
    <item row="0" column="1">
     <widget class="QComboBox" name="EngineerComboBox"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="EngineerLabel">
      <property name="text">
       <string>Кто установил:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>471</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="SerialLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="SerialLabel">
      <property name="text">
       <string>Серийный номер</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="SerialComboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>130</y>
     <width>471</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="CustomerLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="CustomerLabel">
      <property name="minimumSize">
       <size>
        <width>37</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>37</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Клиент&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="CustomerComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>301</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="SoftwareLayout">
    <item row="0" column="0">
     <layout class="QFormLayout" name="VendorLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="VendorLabel">
        <property name="text">
         <string>Производитель:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="VendorComboBox"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QFormLayout" name="SoftTitleLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="SoftLabel">
        <property name="text">
         <string>Программный продукт:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="SoftComboBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>130</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="CountLayout">
    <item>
     <widget class="QLabel" name="CountLabel">
      <property name="text">
       <string>Количество ПК:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="CountSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="CostLayout">
    <item>
     <widget class="QLabel" name="CostLabel">
      <property name="text">
       <string>Стоимость:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="CostEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="TypeLabel">
      <property name="text">
       <string>Тип:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="TypeComboBox"/>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>DialogBtnBox</zorder>
  <zorder>printBtn</zorder>
  <zorder>AddCustomerBtn</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>layoutWidget_2</zorder>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>DialogBtnBox</sender>
   <signal>accepted()</signal>
   <receiver>SellDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DialogBtnBox</sender>
   <signal>rejected()</signal>
   <receiver>SellDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
